clippy_lints/src/lifetimes.rs

```diff
@@ -169,7 +169,7 @@ fn check_fn_inner<'a, 'tcx>(
         span_lint(
             cx,
             NEEDLESS_LIFETIMES,
-            span,
+            decl.output.span(),
```

Comment:
span.with_hi(decl.output.span().hi()),
```suggestion
            span.with_hi(decl.output.span().hi()),
```

Reply:
Hi @flip1995 
Thanks for the suggestion but ever since I rebased with current master I'm not able to build the package. `cargo build` gives me the following error:
```
error[E0463]: can't find crate for `rustc_trait_selection`
  --> clippy_lints/src/lib.rs:55:1
   |
55 | extern crate rustc_trait_selection;
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate

error: aborting due to previous error

For more information about this error, try `rustc --explain E0463`.
error: could not compile `clippy_lints`.
```
What am I missing here?

Reply:
You would need a new rustup (Update from breakage because of rustc internal change).
Don't worry, it seems to be doing in #5333.

Reply:
Or if you don't want to wait for the rustup. Replace master in `setup-toolchain` with ae5b641e9ef3fc5ed2b9944121f75902f639cb0a. Then run the script again.
(The commit number could be looked up in https://rust-lang-nursery.github.io/rust-toolstate/).

Reply:
Just running `setup-toolchain.sh` should be enough. The rustup in #5333 is still waiting on the rust PR to get merged.

Reply:
Thanks. that seemed to work